tests/dem/post_collision_velocity.output

```diff
@@ -1,2 +1,2 @@
 
-DEAL::Coefficient of restitution is 0.9 and the velocity of particle before collision is 0.1, the velocity of particle after collision is: 0.0899723
+DEAL::Coefficient of restitution is 0.9 and the velocity of particle before collision is 0.1, the velocity of particle after collision is: 0.0882268
```

Comment:
Energy conservation is now lower? Did you change the time step?

Reply:
no, I changed nothing. It was strange for me too.